core/src/signal_path/dsp.cpp

```diff
@@ -4,6 +4,10 @@ SignalPath::SignalPath() {
     
 }
 
+SignalPath::~SignalPath() {
+    stopFFT();
```

Comment:
This is not needed, the destructor of each member takes care of termination.

Reply:
On master - yes, I will remove this